js/fileUtils.js

```diff
@@ -9,26 +31,22 @@ export function genMediaTags(file) {
   if (typeof file === "string" && !/^[a-z]+:\/\//i.test(file)) {
     file = `${location.protocol}//${location.host}${location.pathname}${file}`;
   }
-  return new Promise((resolve, reject) => {
-    require.ensure(
-      ["jsmediatags/dist/jsmediatags"],
-      require => {
-        const jsmediatags = require("jsmediatags/dist/jsmediatags");
-        try {
-          jsmediatags.read(file, { onSuccess: resolve, onError: reject });
-        } catch (e) {
-          // Possibly jsmediatags could not find a parser for this file?
-          // Nothing to do.
-          // Consider removing this after https://github.com/aadsm/jsmediatags/issues/83 is resolved.
-          reject(e);
-        }
-      },
-      () => {
-        // The dependency failed to load
-      },
-      "jsmediatags"
-    );
-  });
+  return require.ensure(
+    ["music-metadata"],
+    require => {
+      const mm = require("music-metadata");
+      return sourceToStream(file).then(stream => {
```

Comment:
The rest of the code base uses `async`/`await`. Might as well use it here too.

Reply:
I will adapt to you coding convention, no problem